### PR TITLE
add and refresh authors section

### PR DESCRIPTION
**Pull request by fangohr**
- be more explicit who can be contacted about the page
- and who should be blamed / acknowledged

### CODE REVIEW

File: tools/pelican/content/pages/index.rst [hunk 1]
```diff
@@ -65,6 +65,20 @@ touch <mailto:oscovidaproject@gmail.com>`__. Ideas, suggestions and
 error reports, are welcome at our
 `issue tracker <https://github.com/oscovida/oscovida/issues>`__.
 
+History and authors
+===================
+
+This project was started by `Hans Fangohr <https://fangohr.github.io>`__ early in 2020, and later developed further with the
+`Photon and Neutron Open Science Cloud PaNOSC) <https://www.panosc.eu/>`__ and
```
Comment (kirienko) [inline tools/pelican/content/pages/index.rst:72]:
```suggestion
`Photon and Neutron Open Science Cloud (PaNOSC) <https://www.panosc.eu/>`__ and
```

File: tools/pelican/content/pages/index.rst [hunk 2]
```diff
@@ -76,10 +90,9 @@ processing. You are welcome to use the material at your own risk. See
 Acknowledgements
 ================
 
--  The H2020 project `Photon and Neutron Open Science Cloud
-   PaNOSC) <https://www.panosc.eu/>`__
 -  `Max Planck Institute for the Structure and Dynamics of Matter <https://www.mpsd.mpg.de/en>`__
 -  `European XFEL <http://www.xfel.eu>`__
+-  The H2020 project `Photon and Neutron Open Science Cloud PaNOSC) <https://www.panosc.eu/>`__
```
Comment (kirienko) [inline tools/pelican/content/pages/index.rst:95]:
```suggestion
-  The H2020 project `Photon and Neutron Open Science Cloud (PaNOSC) <https://www.panosc.eu/>`__
```

File: tools/pelican/content/pages/index.rst [hunk 3]
```diff
@@ -65,6 +65,20 @@ touch <mailto:oscovidaproject@gmail.com>`__. Ideas, suggestions and
...
 
+History and authors
+===================
+
+This project was started by `Hans Fangohr <https://fangohr.github.io>`__ early in 2020, and later developed further with the
+`Photon and Neutron Open Science Cloud PaNOSC) <https://www.panosc.eu/>`__ and
+`other volunteers <https://oscovida.github.io/team.html>`__ as an open science demonstrator.
+Currently, the project is maintained by Hans Fangohr from the
+`Max Planck Institute for Structure and Dynamics of Matter <https://www.mpsd.mpg.de/research/ssus/comput-science>`__,
+`Yuri Kirienko <https://github.com/kirienko>`__, and
```
Comment (kirienko) [inline tools/pelican/content/pages/index.rst:76]:
```suggestion
`Yury Kirienko <https://github.com/kirienko>`__, and
```

**Review by kirienko — APPROVED** [review]
There are some minor typos, but apart from that LGTM!
Thanks for maintaining!

**Comment by fangohr** [comment]
Great, thanks @kirienko for feedback and corrections!

**Pull request merged**